caf-audit-service-container/pom.xml

```diff
@@ -62,7 +62,7 @@
         </dependency>
         <dependency>
             <groupId>com.github.cafaudit</groupId>
-            <artifactId>caf-audit</artifactId>
+            <artifactId>caf-audit-service-testing-audit</artifactId>
```

Comment:
I would re-name this artifact as it has 'audit' twice in it. Maybe drop '-audit' from the end.